snakemake_executor_plugin_slurm/__init__.py

```
@@ -730,10 +763,129 @@ def check_slurm_extra(self, job):
-                "parameter. The job name is set by snakemake and must not be "
-                "overwritten. It is internally used to check the stati of the "
-                "all submitted jobs by this workflow."
+                "The --job-name option is not allowed in the 'slurm_extra' parameter. "
+                "The job name is set by snakemake and must not be overwritten. "
+                "It is internally used to check the stati of the all submitted jobs "
+                "by this workflow."
                 "Please consult the documentation if you are unsure how to "
                 "query the status of your jobs."
             )
+
+    def create_efficiency_report(self, efficiency_threshold=0.8):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)